slides/02-data-budget.qmd

```diff
@@ -243,37 +252,33 @@ Make a plot or summary and then share with neighbor
 
 ```{r}
 #| fig-align: 'center'
-ggplot(frog_train, aes(latency)) +
-  geom_histogram(bins = 20)
+ggplot(taxi_train, aes(x = local, fill = tip)) +
+  geom_bar(position = "fill")
 ```
 
-:::notes
-This histogram brings up a concern. What if in our training set we get unlucky and sample few or none of these large values? That could mean that our model wouldn't be able to predict such values. Let's come back to that!
-:::
-
 ## 
 
 ```{r}
 #| fig-align: 'center'
-ggplot(frog_train, aes(latency, treatment, fill = treatment)) +
-  geom_boxplot(alpha = 0.5, show.legend = FALSE)
+ggplot(taxi_train, aes(x = hour, fill = tip)) +
+  geom_bar()
 ```
 
 ## 
 
 ```{r}
 #| fig-align: 'center'
-frog_train %>%
-  ggplot(aes(latency, reflex, fill = reflex)) +
-  geom_boxplot(alpha = 0.3, show.legend = FALSE)
+ggplot(taxi_train, aes(x = hour, fill = tip)) +
+  geom_bar(position = "fill")
 ```
 
 ## 
 
 ```{r}
 #| fig-align: 'center'
-ggplot(frog_train, aes(age, latency, color = reflex)) +
-  geom_point(alpha = .8, size = 2)
+ggplot(taxi_train, aes(x = distance)) +
+  geom_histogram(bins = 20) +
```

Comment:
I think `bins = 20` is way to low. I like to go `bins = 100`. It of cause depends on the data at hand. With these lower bin counts we are losing information about how there are quite a few SUPER short trip

# bins = 20

<img width="903" alt="Screenshot 2023-07-05 at 3 03 44 PM" src="https://github.com/tidymodels/workshops/assets/14034784/7e81d0e8-5277-4569-8d8e-7fc7ac60a047">

# bins = 100

<img width="918" alt="Screenshot 2023-07-05 at 3 01 29 PM" src="https://github.com/tidymodels/workshops/assets/14034784/e2081640-9906-4357-96d8-f084780ab557">

# bins = 1000

<img width="925" alt="Screenshot 2023-07-05 at 3 01 19 PM" src="https://github.com/tidymodels/workshops/assets/14034784/2aad649c-06b7-4f29-a9dd-74c1757402bc">

Reply:
You're right! [90cb6df](https://github.com/tidymodels/workshops/pull/108/commits/90cb6df41f284734cd06baf245eefe9dc6a8b279)